Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,112 @@
+- UIA [ControlType](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value "MenuItem" to `T:System.Windows.Forms.ToolStripMenuItem`
+- UIA [Name](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationelement.nameproperty?view=netframework-4.7) property and Expand/Collapse pattern to `T:System.Windows.Forms.ToolStripItem`
+- [AccessibleEvents](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessibleevents?view=netframework-4.7) indicating StateChange and NameChange when drop down is expanded or collapsed to `T:System.Windows.Forms.ToolStripDropDownItem`
+- UIA ControlType property value "Button" to `T:System.Windows.Forms.ToolStripDropDownButton`
+- UIA [TogglePattern](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.togglepattern?view=netframework-4.7) to `T:System.Windows.Forms.DataGridViewCheckBoxCell`
+- UIA Expand/Collapse pattern to `T:System.Windows.Forms.ComboBox` 
+ 
+__Improvements to PropertyBrowser control__
+- Added UIA Expand/Collapse pattern, state and name change notifications and ControlType property with value "Button" to the "Details" button in the error dialog that is displayed when the user enters incorect value in the `T:System.Windows.Forms.PropertyGrid` control 
+- Changed [AccessibleRole](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessiblerole?view=netframework-4.7) of rows in PropertyGrid control from "Row" to "Cell", which maps to UIA ControlType "DataItem", in order to support appropriate keyboard shortcuts and Narrator announcements
+- Improved keyboard navigation between the grid and the ToolBar above it. Pressing "Shift-Tab" now selects the first ToolBar button, instead of the whole ToolBar
+
+__Ensure that OS-defined colors are used in High Contrast themes__
+
+_Note:_ 
+Windows10 had changed values for some high contrast system colors in the *Fall Creator's Update* release. Windows Forms Framework is based on Win32 framework, for he best experience, please run on the latest version of windows and opt in into the latest OS changes by adding app.manifest file in test application and uncommented following code:
+```
+    <!-- Windows 10 -->
+    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+```
+
+- `T:System.Windows.Forms.Button` and `T:System.Windows.Forms.CheckBox` controls with `P:System.Windows.Forms.Control.FlatStyle` set to `F:System.Windows.Forms.FlatStyle.Standard`, which is the default style, are now using OS-defined colors in High Contrast theme in selected state. Before this change test and background colors were not contrasting and were hard to read
```

Comment:
are now using --> now use
in selected state --> when selected (?)
Before this change --> Previously, (comma after Previously)
period after read.